README.md

```diff
@@ -158,44 +158,75 @@ If the command succeeds you should get a result like this:
 
 Queries are written in the new [Kusto Query Language](https://kusdoc2.azurewebsites.net/docs/query/index.html).
```

Comment:
This URL is not fully public, a better documention URL is probably https://docs.microsoft.com/en-us/azure/kusto/query/